index.html

```diff
@@ -58,135 +78,138 @@ <h2>I’m a software developer</h3>
+                    <div class="body-content">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
+                </li>
+                <li class="project pr-5 bg-img btn-desktop">
+                    <div class="body-content">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
+                </li>
+                <li class="project pr-6 bg-img btn-desktop">
+                    <div class="body-content">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
+                </li>
+                <li class="project pr-7 bg-img btn-desktop">
+                    <div class="body-content">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
+                </li>
+            </ul>
+        </div>
```

Comment:
- [ ] Please note that the first project card should be the same as all the others in the default/intial state , only when the user hovers on any card , does the description and dark opacity should disappears and a See Project button appears at the bottom of the image , kindly fix that 🙏 


![image](https://user-images.githubusercontent.com/89282221/192866147-db6ee1b4-15b2-4afc-b9a1-052c00cd4560.png)

Reply:
Thanks for the observation 🙏🏼 corrections have been made